### DISCUSSION TITLE
Can Karavan easily be integrated into another host React app?

### DISCUSSION THREAD

**Discussion by jeffreyschultz** [apache/camel-karavan#1020] [General]
I am looking at Karavan to possibly make it available as part of a larger system and am curious if it could easily be integrated into another React app.

Has this been considered in its design? Or would there be quite a bit of work to get it to work correctly?

**Comment by mgubaidullin** [comment]
Karavan Designes was not developed with this scenario in mind. However I heard some organizations did that.

**Comment by davsclaus** [comment]
answered by marat
